lxl-web/src/lib/utils/search.ts

```diff
@@ -90,6 +92,17 @@ export function displayMappings(
 			const operator = _hasOperator(m);
 
 			if ('property' in m && operator) {
+				// Mock old behaviour for 'classic' search GUI, i.e show error page
+				// when encountering an invalid property in order to provide feedback.
+				// TODO remove this when Supersearch is fully implemented.
+				const useSuperSearch = env?.PUBLIC_USE_SUPERSEARCH === 'true';
+				const _propertyType = m.property?.['@type'];
+				if (!useSuperSearch && _propertyType === '_Invalid') {
```

Comment:
I would skip the const declaration on the line above and just write:
```js
if (!useSuperSearch && m.property?.['@type'] === '_Invalid') {
```
(as `_propertyType` isn't used anywhere else).

Looks good otherwise!

Reply:
Sure, was trying to fix the typescript complaint and somehow it stayed in there. Fixed.